app/include/zmk/hid.h

```diff
@@ -21,122 +22,119 @@
     /* INPUT (Data,Var,Abs) */
-    HID_MI_INPUT,
-    0x02,
+    HID_MI_INPUT, 0x02,
 
     /* USAGE_PAGE (Keyboard/Keypad) */
-    HID_GI_USAGE_PAGE,
-    HID_USAGE_KEY,
+    HID_GI_USAGE_PAGE, HID_USAGE_KEY,
     /* REPORT_SIZE (8) */
-    HID_GI_REPORT_SIZE,
-    0x08,
+    HID_GI_REPORT_SIZE, 0x08,
     /* REPORT_COUNT (1) */
-    HID_GI_REPORT_COUNT,
-    0x01,
+    HID_GI_REPORT_COUNT, 0x01,
     /* INPUT (Cnst,Var,Abs) */
-    HID_MI_INPUT,
-    0x03,
+    HID_MI_INPUT, 0x03,
 
     /* USAGE_PAGE (Keyboard/Keypad) */
-    HID_GI_USAGE_PAGE,
-    HID_USAGE_KEY,
+    HID_GI_USAGE_PAGE, HID_USAGE_KEY,
     /* LOGICAL_MINIMUM (0) */
-    HID_GI_LOGICAL_MIN(1),
-    0x00,
+    HID_GI_LOGICAL_MIN(1), 0x00,
     /* LOGICAL_MAXIMUM (0xFF) */
-    HID_GI_LOGICAL_MAX(1),
-    0xFF,
+    HID_GI_LOGICAL_MAX(1), 0xFF,
     /* USAGE_MINIMUM (Reserved) */
-    HID_LI_USAGE_MIN(1),
-    0x00,
+    HID_LI_USAGE_MIN(1), 0x00,
     /* USAGE_MAXIMUM (Keyboard Application) */
-    HID_LI_USAGE_MAX(1),
-    0xFF,
+    HID_LI_USAGE_MAX(1), 0xFF,
     /* REPORT_SIZE (1) */
-    HID_GI_REPORT_SIZE,
-    0x08,
+    HID_GI_REPORT_SIZE, 0x08,
     /* REPORT_COUNT (ZMK_HID_KEYBOARD_NKRO_SIZE) */
-    HID_GI_REPORT_COUNT,
-    ZMK_HID_KEYBOARD_NKRO_SIZE,
+    HID_GI_REPORT_COUNT, ZMK_HID_KEYBOARD_NKRO_SIZE,
     /* INPUT (Data,Ary,Abs) */
-    HID_MI_INPUT,
-    0x00,
+    HID_MI_INPUT, 0x00,
 
     /* END_COLLECTION */
     HID_MI_COLLECTION_END,
     /* USAGE_PAGE (Consumer) */
-    HID_GI_USAGE_PAGE,
-    HID_USAGE_CONSUMER,
+    HID_GI_USAGE_PAGE, HID_USAGE_CONSUMER,
     /* USAGE (Consumer Control) */
-    HID_LI_USAGE,
-    HID_USAGE_CONSUMER_CONSUMER_CONTROL,
+    HID_LI_USAGE, HID_USAGE_CONSUMER_CONSUMER_CONTROL,
     /* Consumer Page */
-    HID_MI_COLLECTION,
-    COLLECTION_APPLICATION,
+    HID_MI_COLLECTION, COLLECTION_APPLICATION,
     /* REPORT ID (1) */
-    HID_GI_REPORT_ID,
-    0x02,
+    HID_GI_REPORT_ID, 0x02,
     /* USAGE_PAGE (Consumer) */
-    HID_GI_USAGE_PAGE,
-    HID_USAGE_CONSUMER,
+    HID_GI_USAGE_PAGE, HID_USAGE_CONSUMER,
     /* LOGICAL_MINIMUM (0) */
-    HID_GI_LOGICAL_MIN(1),
-    0x00,
+    HID_GI_LOGICAL_MIN(1), 0x00,
     /* LOGICAL_MAXIMUM (0xFFFF) */
-    HID_GI_LOGICAL_MAX(2),
-    0xFF,
-    0xFF,
-    HID_LI_USAGE_MIN(1),
-    0x00,
+    HID_GI_LOGICAL_MAX(2), 0xFF, 0xFF, HID_LI_USAGE_MIN(1), 0x00,
     /* USAGE_MAXIMUM (0xFFFF) */
-    HID_LI_USAGE_MAX(2),
-    0xFF,
-    0xFF,
+    HID_LI_USAGE_MAX(2), 0xFF, 0xFF,
     /* INPUT (Data,Ary,Abs) */
     /* REPORT_SIZE (16) */
-    HID_GI_REPORT_SIZE,
-    0x10,
+    HID_GI_REPORT_SIZE, 0x10,
     /* REPORT_COUNT (ZMK_HID_CONSUMER_NKRO_SIZE) */
-    HID_GI_REPORT_COUNT,
-    ZMK_HID_CONSUMER_NKRO_SIZE,
-    HID_MI_INPUT,
-    0x00,
+    HID_GI_REPORT_COUNT, ZMK_HID_CONSUMER_NKRO_SIZE, HID_MI_INPUT, 0x00,
     /* END COLLECTION */
     HID_MI_COLLECTION_END,
+
+    0x05, 0x01,                                 /* Usage Page (Generic Desktop Ctrls) */
```

Comment:
are there macros for these constants, so it's more readable similar to the rest of the HID report?

Reply:
There should be, but I'm not sure the defines are in the files right now. I was planning to add them later.

Reply:
Added them in, everything still works.